java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -382,6 +384,37 @@ public T getdel(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Gets the value associated with the given <code>key</code>.
+     *
+     * @since Redis 6.2.0.
+     * @see <a href="https://redis.io/docs/latest/commands/getex/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return Command Response - If <code>key</code> exists, return the value of the <code>key</code>
+     *     . Otherwise, return <code>null</code>.
+     */
+    public T getex(@NonNull String key) {
+        ArgsArray commandArgs = buildArgs(key);
+        protobufTransaction.addCommands(buildCommand(GetEx, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Gets the value associated with the given <code>key</code>.
+     *
+     * @since Redis 6.2.0.
+     * @see <a href="https://redis.io/docs/latest/commands/getex/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @param options The {@link GetExOptions} options.
+     * @return Command Response - If <code>key</code> exists, return the value of the <code>key</code>
+     *     . Otherwise, return <code>null</code>.
```

Comment:
if you want you can maybe change the wording to something like  "return the value stored at <code>key</code>" to see if spotless does a less ugly line break

Reply:
fixed